dashboard/test/factories/pd_factories.rb

```diff
@@ -49,6 +52,57 @@
         workshop.enrollments << build(:pd_enrollment, workshop: workshop, user: teacher)
       end
     end
+
+    after(:create) do |workshop, evaluator|
+      workshop.sessions.map(&:save)
+
+      evaluator.num_facilitators.times do
+        workshop.facilitators << (create :facilitator)
+      end
+
+      evaluator.num_completed_surveys.times do
+        enrollment = create :pd_enrollment, workshop: workshop
+        if workshop.teachercon?
+          survey_hash = build :pd_teachercon_survey_hash
+
+          Pd::TeacherconSurvey.facilitator_required_fields.each do |field|
+            survey_hash[field] = {}
+          end
+
+          survey_hash['whoFacilitated'] = workshop.facilitators.map(&:name)
+
+          workshop.facilitators.each do |facilitator|
+            Pd::TeacherconSurvey.facilitator_required_fields.each do |field|
+              survey_hash[field][facilitator.name] = 'Free response'
+            end
+          end
+
+          create :pd_teachercon_survey, pd_enrollment: enrollment, form_data: survey_hash.to_json
```

Comment:
Having second thoughts on moving this.

For testing purposes, I'm usually going to be saying "Give me a workshop with a bunch of enrollments and survey responses associated with those enrollments." With the current setup, that's one line with this statement. With what you're proposing, it would instead be
```create the workshop
create the enrollments
create the surveys that use workshop data to fill out their contents```

I'm not really sure if that's better.

If this isn't a big deal, I'd like to punt on this for the time being because of the time pressure.